### DISCUSSION TITLE
ERROR - Failed to import account: maFile missing field `steam_id` at line 1 column 777

### DISCUSSION THREAD

**Discussion by Aurora15304** [dyc3/steamguard-cli#230] [Q&A] [answered]
![image](https://github.com/dyc3/steamguard-cli/assets/95676130/ff16b34b-0c23-4427-9f8d-ce49c8928285)

**Comment by dyc3** [comment]
The import command doesn't have the capability to tell if the file you are importing is an SDA one, or a steamguard cli one. If it's an SDA one, add the `--sda` argument to your command

**Reply by Aurora15304** [reply]
Oh no I still won't import my mafile into your project

**Reply by dyc3** [reply]
I'm confused, can you elaborate a little more?

**Reply by Aurora15304** [reply]
How to import the generated maflies file into your steamguard-cli and use

**Reply by dyc3** [reply]
You gotta give me a little more information here. Have you read the quickstart? https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md

Is the maFile from SDA or steamguard-cli? Do you have other accounts that work already with steamguard-cli?

**Reply by Aurora15304** [reply]
![image](https://github.com/dyc3/steamguard-cli/assets/95676130/e5f88171-64fe-449e-8baa-507665488d8b)
mafile comes from SDA but why can't I import it
Attached is the command I used

**Reply by dyc3** [reply, marked as answer]
Try this:
```
.\steamguard import --sda --files %APPDATA%\steamguard-cli\maFiles\123.maFile
```

**Reply by Aurora15304** [reply]
ok  is work
